priv/templates/simplenode.reltool.config

```diff
@@ -38,7 +38,7 @@
            {copy, "files/{{nodeid}}", "bin/{{nodeid}}"},
            {copy, "files/{{nodeid}}.cmd", "bin/{{nodeid}}.cmd"},
            {copy, "files/start_erl.cmd", "bin/start_erl.cmd"},
-           {copy, "files/install_upgrade.escript", "bin/install_upgrade.escript"},
+           {copy, "files/install_upgrade.escript", "bin/install_upgrade.escript"},  % may be safely removed in new projects
```

Comment:
Thanks, the new version has to use `%%` though, and it should be indented like the surrounding lines.